apps/example/index.tsx

```diff
@@ -76,6 +76,13 @@ const App = createApp({
     protocolIds: ['somm'],
     referrerId: 'some-referrer-id',
   },
+  themes: {
+    default: {
+      assets: {
+        welcomeBackgroundImage: undefined,
+      },
+    },
+  },
```

Comment:
I think the default shouldn't need this and should be a Divvi branded background to work as an example.
What do you think?